packages/lab/src/overlay/OverlayTrigger.tsx

```diff
@@ -21,17 +21,10 @@ export function OverlayTrigger(props: OverlayTriggerProps) {
     return <>{children}</>;
   }
 
-  const getTriggerProps = () =>
-    getReferenceProps({
-      ref: reference,
-      id: `${id}-trigger`,
-      "aria-controls": `${id}-panel`,
```

Comment:
Is aria-controls needed here ? 
Reference ref is already passed down in trigger ref and id is not needed